### PR TITLE
Added a list of built-in events

### PR DESCRIPTION
**Pull request by jtgrimes**
Also updated several components to indicate what events they fire.

### CODE REVIEW

**Comment by gmsantos** [comment]
I think its a good addition to docs. Is there any issue with this PR @taylorotwell ?

**Comment by taylorotwell** [comment]
I can't find these cached events in the source. Am I having a brain fart? :)

File: events.md [hunk 1]
```diff
@@ -431,3 +432,29 @@ Once the subscriber has been defined, it may be registered with the event dispat
...
+connection.{name}.committed | $connection
+connection.{name}.rollingBack | $connection
+illuminate.query | $query, $bindings, $time, $connectionName
+illuminate.queue.after | $connection, $job, $data
+illuminate.queue.failed | $connection, $job, $data
+illuminate.queue.stopping | null
+mailer.sending | $message
+router.matched | $route, $request
+composing:{view name} | $view
+creating:{view name} | $view
```
Comment (crynobone) [inline events.md]: I think there is a space between `composing: {view}` and `creating: {view}`.

**Comment by crynobone** [comment]
> I can't find these cached events in the source. Am I having a brain fart? :)

https://github.com/laravel/framework/blob/5.1/src/Illuminate/Cache/Repository.php#L73

**Pull request merged**